src/cloudant/database.py

```diff
@@ -18,9 +18,17 @@
 import json
 import contextlib
 import posixpath
-import urllib
+
 from requests.exceptions import HTTPError
 
+# pylint: disable=wrong-import-order
+from ._2to3 import PY2
+if PY2:
+    # pylint: disable=wrong-import-order,no-name-in-module
+    from urllib import quote_plus
+else:
+    from urllib.parse import quote_plus  # pylint: disable=import-error,no-name-in-module
```

Comment:
Can you put the pylint disable comment above the statement?  This more closely matches the coding standard that we've adopted.